BookReader/BookReader.css

```diff
@@ -507,6 +507,7 @@ body.BRfullscreenActive .mm-menu {
 }
 .BRpagecontainer.BRemptypage {
   background: transparent;
+  display: none;
```

Comment:
we will isolate change to text-selection layer in order to preserve wrapper size & state.

Reply:
pushed changes on this [on this commit](https://github.com/internetarchive/bookreader/pull/592/commits/8f0305c92884686ee9bb89314355cce8c728841a)

Reply:
perfect ✔️